common/Configuring_CloudForms.adoc

```diff
@@ -0,0 +1,155 @@
+[[Configuring-cloudforms]]
+== Configuring {product-title}
```

Comment:
Top level headers should be L1 (one '=' only):

    = Configuring {product-title}